util/changePrerelease

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env sh
+
+#Script for removing pre-release tag and updating version number before creating release branch. Arguments are the old and new version numbers as follows:
+#1 - Old Major
+#2 - New Major
+#3 - Old Minor
+#4 - New Minor
+#5 - Old Update
+#6 - New Update
+
+sed -i '' 35s/false/true/ ../../compiler/main/version_num.h
```

Comment:
Explicit line numbers are an unreliable way to locate the changes. The `sed` command here will silently fail, making no changes to the file, if someone adds or removes any lines before the current line 35. We're also relying here on the current working directory being two dirs below `$CHPL_HOME`, which seems unnecessary and/or undesirable. Preferable might be something like:

```
sed -i '/ officialRelease = /s/false/true/' $CHPL_HOME/compiler/main/version_num.h
```

This uses `/regexp/` style line addressing to make the change on the specific variable declaration we're aiming for, and locates the file based on `$CHPL_HOME`.

Also, what's the `''` command line argument after the `-i` in the original command? I'm not seeing anything that matches up with that in the `sed(1)` man page (on cflcs01).

While here, I'll predict that @ronawho and possibly others will recommend you do this in python instead of the shell. 😁

Reply:
These are all very good thank you I'll go implement these.

The `-i ''` is a Mac thing. On Mac sed -i needs a blank argument to actually make changes in the file itself. Since we'll be running these locally on Macs they'll need to be kept in there. This info came from this SO post: https://stackoverflow.com/questions/7573368/in-place-edits-with-sed-on-os-x

Reply:
Oh, that's a shame. I think we'd like to be able to run it on either Mac or Linux, though, wouldn't we? Left to my own devices I might use `-i.backup` and then delete the backup file after checking for success in the command, as in the following (expressed in two lines for clarity):

```
f=$CHPL_HOME/compiler/main/version_num.h
sed -i.backup '/ officialRelease = /s/false/true/' $f && rm $f.backup
```

This might actually be another reason to use python, though.